api/utils/bool.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // ParseBool parses string as boolean value,
-// returns error in case if value is not recognized
+// returns false, error in case if value is not recognized
 func ParseBool(value string) (bool, error) {
 	switch strings.ToLower(value) {
 	case "yes", "yeah", "y", "true", "1", "on":
```

Comment:
Why do we have `nope` but no `yup` I wonder 😄

Reply:
```diff
+ yeppers
```

In all seriousness though, this is a fairly well-trodden path in the Teleport codebase already... 😅 

https://github.com/gravitational/teleport/blob/5a65ba9e7323e15769c9e3abb5d3908c4abb8344/api/types/authentication.go#L894-L897

https://github.com/gravitational/teleport/blob/5a65ba9e7323e15769c9e3abb5d3908c4abb8344/api/utils/bool.go#L30-L33

Reply:
Also my personal favourite

https://github.com/gravitational/teleport/blob/5a65ba9e7323e15769c9e3abb5d3908c4abb8344/lib/config/configuration_test.go#L1433